File: Java-Core/src/main/java/presto/python/Python3NativeCall.java
package presto.python;

import presto.utils.CodeWriter;

public class Python3NativeCall extends PythonNativeCall {

	public Python3NativeCall(PythonStatement statement) {
		super(statement);
	}

	@Override
	public void toDialect(CodeWriter writer) {
		writer.append("Python3: ");
		super.toDialect(writer);
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/python/Python3NativeCall.js
var PythonNativeCall = require("./PythonNativeCall").PythonNativeCall;

function Python3NativeCall(statement, module) {
	PythonNativeCall.call(this, statement, module);
	return this;
}

Python3NativeCall.prototype = Object.create(PythonNativeCall.prototype);
Python3NativeCall.prototype.constructor = Python3NativeCall;

Python3NativeCall.prototype.toDialect = function(writer) {
    writer.append("Python3: ");
    PythonNativeCall.prototype.toDialect.call(this, writer);
};


exports.Python3NativeCall = Python3NativeCall;